exampleSite/content/index/footer.md

```diff
@@ -6,10 +6,13 @@ lastmod = "2017-09-07"
 weight = 1200
 #background = ""
 
-link_title = "Link Title"
-logo_title = "Logo Title"
-logo = "logo.svg"
-logo_text = "Logo Subtext"
+[[link]]
+    title = "Link Title"
+
+[[logo]]
```

Comment:
Sidenote. Should we give an option to link it? Or does that feel unnecessary? We can open an issue to discuss this further I assume.

Reply:
Link the title? or the links beneath it. if the latter is the case I think is already a link. Might need some clearifications

Reply:
Link the logo itself. Similar on how we do that with the logo in the navigation bar or within the logo fragment.

Reply:
Oh right. I think we should. It makes sense to have it linked too like the other logo.